### ISSUE TITLE
Request: Update to MSAL

### ISSUE THREAD

**Issue by HiltonGiesenow** [OfficeDev/msteams-tabs-sso-sample-nodejs#9]
Thanks for the great sample, really helping me figure this out. I assume it's possible to use MSAL instead of ADAL though? If so, would it be possible to update this sample?

**Comment by marwalsch** [comment]
I have been redirected here from two different [threads](https://stackoverflow.com/questions/63388414/msal-sso-with-microsoft-teams-tabs) to here claiming that this is the official sample for Teams SSO with MSAL, though this is clearly not the case. Teams docs for migrating are somewhat lacking, so I'd highly appreciate this.

**Comment by HeinA** [comment]
Yes.  Trying to use modern authentication is teams is a complete nightmare.  
It would be great to see how to get SSO working with MSAL
